packages/wallets/package.json

```diff
@@ -28,12 +28,14 @@
         "react-router-dom": "^5.2.0",
         "react-transition-group": "4.4.2",
         "usehooks-ts": "^2.7.0",
-        "yup": "^0.32.11"
+        "yup": "^0.32.11",
+        "zxcvbn": "^4.4.2"
```

Comment:
I am not sure if we can afford the size of this bundle - cc @yashim-deriv @markwylde-deriv 

https://bundlephobia.com/package/zxcvbn@4.4.2

<img width="536" alt="Screenshot 2023-11-22 at 08 59 52" src="https://github.com/binary-com/deriv-app/assets/141034155/3d6c9ea1-5d75-4b34-a93d-1d895cf30e2c">

Reply:
Yeah I pointed this out here too, but the PR was closed. Not sure why:
https://github.com/binary-com/deriv-app/pull/11621#discussion_r1400503588

At a minimum we should use the same `@contentpass/zxcvbn` as the other libraries, but I agree @wojciech-deriv this doesn't look like a well supported, lightweight, library and I'd rather we didn't use it at all.

Reply:
I agree. @aizad-deriv find an alternative for this. This is too huge

Reply:
I found an alternative to this https://bundlephobia.com/package/@zxcvbn-ts/core@3.0.4
Will try and implement it